src/System.ComponentModel.TypeConverter/tests/Drawing/ColorConverterTests.cs

```diff
@@ -446,5 +447,21 @@ public void GetStandardValuesExclusive()
             var conv = new ColorConverter();
             Assert.False(conv.GetStandardValuesExclusive());
         }
+
+        [Fact]
+        public void ConvertToStringTests()
+        {
+            var conv = new ColorConverter();
+            Assert.Equal("Blue", conv.ConvertTo(Color.Blue, typeof(string)));
+            Assert.Equal("ActiveCaption", conv.ConvertTo(SystemColors.ActiveCaption, typeof(string)));
+        }
+
+        [Fact]
+        public void ConvertToInstanceDescriptorTests()
+        {
+            var conv = new ColorConverter();
+            Assert.NotNull(conv.ConvertTo(Color.Blue, typeof(InstanceDescriptor)));
+            Assert.NotNull(conv.ConvertTo(SystemColors.ActiveCaption, typeof(InstanceDescriptor)));
```

Comment:
Maybe we could invoke the property to get the value and compare it here to make sure we got the right property?

Reply:
how about something like this 
```c#
            InstanceDescriptor t = (InstanceDescriptor)conv.ConvertTo(SystemColors.ActiveCaption, typeof(InstanceDescriptor));
            Assert.Equal("ActiveCaption", t.MemberInfo.Name);
```